docs/src/gravity_wave.md

```diff
@@ -0,0 +1,46 @@
+# Gravity wave parameterization
+Gravity waves have great impact on the atmospheric circulation. They are usually generated from topography or convection, propagate upward and alter temperature and winds in the middle atmosphere, and influence tropospheric circulation through downward control. The horizontal wavelength for gravity waves ranges from several km's to hundreds of km's, which is smaller than typical GCM resolution and needs to be parameterized.
+
+## Non-orographic gravity wave
+NGWD parameterization follows the spectra methods described in [alexander1999](@cite). The following assumptions are made for this parameterization to work:
+* The wave spectrum consists of independent monochromatic waves, and wave-wave interaction is neglected when propagation and instability is computed.
+* The gravity wave propagates vertically and conservatively to the breaking level and deposits all momentum flux into that level, as opposed to the method using saturation profile described in [lindzen1981](@cite).
+* The wave breaking criterion is derived in a hydrostatic, non-rotating frame. Including nonhydrostatic and rotation proved to be have neglegible impacts.
+* The gravity wave is intermittent and the intermittency is computed as the ratio of the total long-term average source to the integral of the source momentum flux spectrum.
+
+### Spetrum of the momentum flux sources
+The source spectrum with respect to phase speed is prescribed in Eq. (17) in [alexander1999](@cite). We adapted the equation so that the spectrum writes as a combination of a wide and a narrow bands as
+```math
+B_0(c) = \frac{F_{S0}(c)}{\rho_0} = sgn(c-u_0) \left( Bm\_w \exp\left[ -\left( \frac{c-c_0}{c_{w\_w}} \right) \ln{2} \right] + Bm\_n \exp\left[ -\left( \frac{c-c_0}{c_{w\_n}} \right) \ln{2} \right] \right)
+```
+where the subscript ``0`` denotes values at the source level. ``c_0`` is the phase speed with the maximum flux magnitude ``Bm``. ``c_w`` is the half-width at half-maximum of the Gaussian.  ``\_w`` and ``\_n`` represent the wide and narrow bands of the spectra.
```

Comment:
```suggestion
where the subscript ``0`` denotes values at the source level. ``c_0`` is the phase speed with the maximum flux magnitude ``Bm``. ``c_w`` is the half-width at half-maximum of the Gaussian.  Subscripts w and n represent the wide and narrow bands of the spectra.
```

Reply:
It is actually `_w`/`_n` in the equation to differentiate the preceding subscribes of `0` or `m`

Reply:
Ah, ok!